in.bharatkalluri.splash.json

```diff
@@ -0,0 +1,36 @@
+{
+    "app-id" : "in.bharatkalluri.splash",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "splash",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+
+        "--filesystem=~/.cache",
+        "--filesystem=~/Pictures",
+
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
```suggestion
```

Reply:
Splash needs to talk to dconf (`org.gnome.desktop.background` gsetting) to change the wallpaper. If I drop these lines, I think I won't be able to change the wallpaper.

Reply:
You can use the wallpaper portal for that

Reply:
Not a blocker though, as there's no easy and nice way to do that without using d-bus calls yourself.

Reply:
I see, in that case let us merge this if everything else looks okay to you. I will create an issue for this in my repo.